### DISCUSSION TITLE
Seurat to loupe browser file

### DISCUSSION THREAD

**Discussion by claradelarosa96** [satijalab/seurat#6932] [Q&A]
Hello,

I have a Seurat object that has already been filtered, processed, etc, after cellranger. I would like to be able to generate a .cloupe file to visualize with the loupe browser. Is there any function in teh Seurat package that does it? If not, do you have any ideas?

Thanks a lot!

**Comment by mcrewcow** [comment]
Hi claradelarosa96,

Usually, upon utilizing cellranger count, the .cloupe file will be also generated. This file you can load into the browser, it is stored in outs subfolder (see screenshot). So basically you don't need Seurat to work in Loupe.
![image](https://user-images.githubusercontent.com/77118598/219488414-a07eff6a-b7aa-43b3-95d2-9d8b88d9a502.png)

However, if you plan to work with the Seurat processed data, this issue may be helpful: https://github.com/satijalab/seurat/issues/1391

**Reply by claradelarosa96** [reply]
Yes, I know cellranger generates cloupe files. However, I have done all the filtering, quality controls etc in Seurat combining different cellranger experiments. So I don't want to use the cellranger cloupe files because they are different from the current data I'm working on. #1391 also doesn't full answer what I want to do, because I don't care primarily about the UMAP projection, but rather the data to upload, which is processed. So, I think I would really need a cloupe file generated from my current rds? But I guess Seurat can't convert rds to cloupe? 
Thanks a lot for your help! :)

**Comment by Dragonmasterx87** [comment]
You can export UMAP coords and map to cloupe browser. Follow the tutorial here: [cloup projection mapping inter-operability](https://support.10xgenomics.com/single-cell-gene-expression/software/visualization/latest/tutorial-interoperability#projections)

Tag me in your reply if you need help extracting UMAP/tSNE coordinates.

**Comment by esiegel** [comment]
The recently released `loupeR` package can be used to convert a Seurat object to a cloupe file.  It currently only supports Gene Expression data.  Installation and usage details can be found [here](https://github.com/10XGenomics/loupeR)
